src/main/kotlin/dev/arbjerg/ukulele/command/RepeatCommand.kt

```diff
@@ -6,15 +6,24 @@ import dev.arbjerg.ukulele.jda.CommandContext
 import org.springframework.stereotype.Component
 
 @Component
-class RepeatCommand : Command ("repeat", "loop", "r") {
-    override suspend fun CommandContext.invoke() {
-        player.isRepeating = !player.isRepeating
-        if (player.isRepeating) {reply("Repeating is now enabled.")}
-        else{reply("Repeating is now disabled.")}
-    }
+class RepeatCommand : Command ("repeat", "r", "repeatone", "ro") {
+   override suspend fun CommandContext.invoke() {
+       if (argumentText.isBlank()) {
+           player.toggleRepeatOne()
+       } else {
+           player.repeatOne = argumentText.toBoolean()
+       }
+       
+       if (player.repeatOne) return reply("The current track will be repeated until toggled off or skipped.")
+       reply("Repeat (one) has been turned off and the queue will proceed as normal.")
+   }
```

Comment:
The old Repeat command dealt with queue looping so it was moved to Loop command.  Now, Repeat command represents Repeat One functionality that repeats the currently playing track (vs looping over the entire queue like the Loop command does).